build_runner_core/lib/src/environment/io_environment.dart

```diff
@@ -86,6 +88,11 @@ class IOEnvironment implements BuildEnvironment {
     }
     return buildResult;
   }
+
+  @override
+  Future<BuildScriptUpdates> buildScriptUpdates(PackageGraph packageGraph, AssetGraph graph) {
```

Comment:
We should instead just `extend BuildEnvironment` to pick up the default impl I think